pages/record/_.vue

```diff
@@ -212,10 +226,23 @@
     },
     async mounted() {
       this.relatedEntities = await searchEntities(this.europeanaEntityUris, { wskey: process.env.EUROPEANA_ENTITY_API_KEY });
+
+      if (process.browser) {
+        if (localStorage.extendedMetadataIsOpen && JSON.parse(localStorage.extendedMetadataIsOpen)) {
```

Comment:
clicking on the button throws an error for me:
`JSON.parse: unexpected character at line 1 column 1 of the JSON data` in firefox
`Unexpected token u in JSON at position 0` in chrome

Reply:
@lbiedinger I have added an additional check.